src/main/java/io/luna/Server.java

```diff
@@ -78,7 +78,7 @@ public void init() throws Exception {
         initPlugins();
         initGame();
 
-        launchPool.shutdown();
+        //launchPool.shutdown();
```

Comment:
Could I get you to uncomment this line before I merge this PR? I have a local proper fix for the callback task being rejected